packages/components/src/components/domainproperties/dataset/DatasetPropertiesPanelFormElements.tsx

```diff
@@ -0,0 +1,297 @@
+                            placeholder="Select dataset category"
+                            onChange={onCategoryChange}
+                            value={categoryId}
+                            options={availableCategories}
+                        />
+                    </Col>
+
+                </Row>
+
+                <DatasetSettingsInput
+                    name="label"
+                    label="Label"
+                    helpTip={labelTip}
+                    value={label}
+                    disabled={false}
+                    onValueChange={onInputChange}
+                    showInAdvancedSettings={false}
+                />
+
+            </Col>
+        );
+    }
+}
+
+interface DataRowUniquenessElementsProps {
+    onRadioChange: (evt: any) => any;
+    dataRowSetting: number;
+}
+
+class DataRowUniquenessElements extends React.PureComponent<DataRowUniquenessElementsProps> {
+    render() {
+        const { onRadioChange, dataRowSetting } = this.props;
+        const radioName = "dataRowSetting";
+
+        return (
+            <div className='dataset_data_row_uniqueness_container'>
+                <Radio
+                    name={radioName}
+                    value={0}
+                    checked={dataRowSetting == 0}
+                    onChange={onRadioChange}
+                >
+                    Participant ID only (demographic data)
+                </Radio>
+
+                <Radio
+                    name={radioName}
+                    value={1}
+                    checked={dataRowSetting == 1}
+                    onChange={onRadioChange}
+                >
+                    Participant ID and timepoint
+                </Radio>
+
+                <Radio
+                    name={radioName}
+                    value={2}
+                    checked={dataRowSetting == 2}
+                    onChange={onRadioChange}
+                >
+                    Participant ID, timepoint, and additional key field
+                </Radio>
+            </div>
+        );
+    }
+}
+
+interface DataRowUniquenessContainerProps {
+    model: DatasetModel;
+    onRadioChange: (e: any) => any;
+    dataRowSetting: number;
+    showAdditionalKeyField: boolean;
+}
+
+export class DataRowUniquenessContainer extends React.PureComponent<DataRowUniquenessContainerProps> {
+    render() {
+        const { model, dataRowSetting, showAdditionalKeyField } = this.props;
+        const domain = model.domain;
+
+        const showAdditionalKeyFieldCls = showAdditionalKeyField ? "dataset_data_row_uniqueness_keyField_show" : "dataset_data_row_uniqueness_keyField_hide";
+
+        return (
+            <>
+                <Col md={6}>
+                    <BasicPropertiesTitle
+                        title="Data Row Uniqueness"
+                    />
+
+                    <div>
+                        Choose criteria for how participants and visits are paired with or without an additional data column
```

Comment:
We should check with MattY. Looking at this in the UI, I wonder if we should put this text into a help tip instead of having it on the page / panel directly.